src/CodeGeneration/ApiGenerator/RestSpecification/XPack/MachineLearning/xpack.ml.revert_model_snapshot.patch.json

```diff
@@ -0,0 +1,11 @@
+{
+  "xpack.ml.revert_model_snapshot": {
+    "url": {
+      "parts": {
+        "snapshot_id": {
+          "required": true
```

Comment:
I opened elastic/x-pack-elasticsearch#2641 so that you can avoid this patch.  Like the other one, I guess you'll need to keep the patch for compatibility with 5.x versions prior to 5.6.3, but hopefully you can simplify things for the 6.x client.

Reply:
Thanks @droberts195. We'll be able to remove the patch when the 5.x client bumps to a Rest API spec version that includes https://github.com/elastic/x-pack-elasticsearch/pull/2641